cita-executor/src/postman.rs

```diff
@@ -305,7 +305,7 @@ impl Postman {
                 for proto_block in sync_res.take_blocks().into_iter() {
                     let open_block = OpenBlock::from(proto_block);
                     if !self.backlogs.insert_synchronized(open_block) {
-                        return false;
+                        continue;
```

Comment:
Could you add some more description for this PR?
What the problem is if `return false` here.

And can we add a test to reproduce this problem?

Reply:
you can try when node0's height is 21, node1 sync block from node0.
when node1's height reach 20, network always feed executor [20,21] block,but inserting block 20 return false,so block 21 can't be executed

Reply:
> when node1's height reach 20, network always feed executor [20,21] block

Why `network` doesn't feed `executor` `[21]` when retry?

I am curious what's preventing adding a test to reproduce this problem?

Reply:
Another question: why `21` is special?

Reply:
that is another question. here,why not deal with next block, just because of prev block duplicated